docs/data/charts/bars/HorizontalBars.tsx

```diff
@@ -0,0 +1,118 @@
+import * as React from 'react';
+import { BarChart } from '@mui/x-charts/BarChart';
+import { axisClasses } from '@mui/x-charts';
```

Comment:
Could we maybe import this from `@mui/x-charts/ChartsAxis`?
Getting back to the strive of avoiding root package imports on our end. 🤔

Reply:
removed due to the previous comment